src/Compress/MimeTypeArchiveAdapterResolver.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Filter\Compress;
+
+use Laminas\Filter\Exception\RuntimeException;
+use Laminas\Filter\File\FileInformation;
+
+use function sprintf;
+
+final class MimeTypeArchiveAdapterResolver implements ArchiveAdapterResolverInterface
```

Comment:
Map the detected mime-type of a file to an archive adapter - the default method of adapter selection